cartesian_impedance_msgs/msg/MaxPathDeviation.msg

```diff
@@ -0,0 +1,12 @@
+# parameters for Maximum allowed Path Deviation from goal when in Cartesian Impedance
+# the values are divided into translational and rotational
+# each of the translational and rotational components is defined as Vector3
+# for more info on Vector3 messages please see the following link:
+# http://docs.ros.org/api/geometry_msgs/html/msg/Vector3.html
+
+
+# maximum deviation from set goal in mm
+geometry_msgs/Vector3 translational
```

Comment:
Should this be `translation` (reads better at translation error), as opposed to stiffness, which reads better as transnational stiffness.